homeassistant/helpers/collection.py

```diff
@@ -619,6 +631,56 @@ async def ws_create_item(
         except ValueError as err:
             connection.send_error(msg["id"], websocket_api.ERR_INVALID_FORMAT, str(err))
 
+    @callback
+    def ws_subscribe(
```

Comment:
Same for this function

```suggestion
    def _ws_subscribe(
```